app/jobs/rss_notifications_checker.rb

```diff
@@ -0,0 +1,59 @@
+require 'rss'
+require 'date'
+
+class RssNotificationsChecker < ApplicationJob
+  queue_as :notifications
```

Comment:
We can also assign executors for each queues if we care about it. E.g: maybe the notifications queue is less important than the pulp_sync queue